api/tests/test_validators.py

```diff
@@ -38,7 +39,7 @@
     "description": "short description",
     "license": "CC0",
     "is_periodic": False,
-    "time_density": None,
+    "time_density": "null",
```

Comment:
Is this supposed to be invalidated? I thought we fixed the schema so that we can properly handle python None is loaded as json null (instead of string null).